.github/workflows/ci.yml

```diff
@@ -102,6 +102,10 @@ jobs:
         run: |
           docker run --rm "$CORE_CI_IMAGE" bash -c "cd /opt/npm_and_yarn && npm run lint"
           docker run --rm "$CORE_CI_IMAGE" bash -c "cd /opt/npm_and_yarn && npm test"
+      - name: Run bundler v1 native helper specs
+        if: matrix.suite == 'bundler'
+        run: |
+          docker run --rm "$CORE_CI_IMAGE" bash -c "cd /opt/bundler/v1 && bundle exec rspec spec"
```

Comment:
We probably want to explicitly set `BUNDLER_VERSION=1 bundle exec rspec spec` to make sure we haven't installed the gems with a different bundler version